tests/isomorphism_problem.py

```diff
@@ -105,11 +105,11 @@
             print('---------------------------')
 
             start_isomorph = time()
-            boolean_isomorph = are_isomorph(G, H)
+            boolean_isomorph = are_isomorph(G.copy(), H.copy())
```

Comment:
In welke gevallen heb je deze copy nodig ?
[ Voor torus24 en products72 passen de test zonder .copy(). ]

Reply:
Omdat je hierna de test aanroept om het aantal isomorphismen te tellen. En nu we nog geen preprocessing hebben gaat het prima. Maar ik heb het ook getest als de twin_removal aanstaat en dan gaat het mis. Omdat je in deze regel G en H doorgeeft, in deze methode de twins worden verwijderd en vervolgens je de grafen waarin de twins al zijn verdwenen doorgeeft aan de count_isomorphisms methode. En daar gaat het dan mis omdat hij daar als initiele graaf de graaf zonder twins neemt.

Reply:
oke top !